subprojects/core/src/main/java/org/gradle/api/internal/tasks/TaskInputFilePropertyBuilderInternal.java

```diff
@@ -20,13 +20,16 @@
 import org.gradle.api.tasks.FileNormalizer;
 import org.gradle.api.tasks.PathSensitivity;
 import org.gradle.api.tasks.TaskInputFilePropertyBuilder;
+import org.gradle.internal.fingerprint.Normalizer;
 
 @NonNullApi
 public interface TaskInputFilePropertyBuilderInternal extends TaskInputFilePropertyBuilder, TaskFilePropertyBuilderInternal {
 
     @Override
     TaskInputFilePropertyBuilderInternal withNormalizer(Class<? extends FileNormalizer> normalizer);
```

Comment:
Maybe we can get rid of `withNormalizer(Class)` here?

Reply:
We can, but not in this PR, as it's a public API modification.